test/setup.js

```diff
@@ -1,14 +1,13 @@
-require('dotenv').config()
-
-beforeEach(async () => {
-  // Set reference to server in order to close the server during teardown.
-  const createServer = require('../app/server')
-  jest.setTimeout(10000)
-  const server = await createServer()
-  await server.initialize()
-  global.__SERVER__ = server
-})
+process.env.EVENT_QUEUE_ADDRESS = 'ffc-ahwr-test'
+process.env.APPLICATION_API_URI = 'http://localhost'
+process.env.CLAIM_SERVICE_URI = 'http://localhost:3004/claim'
+process.env.ENDEMICS_ENABLED = 'true'
+process.env.MESSAGE_QUEUE_HOST = 'test'
+process.env.AZURE_STORAGE_APPLICATION_CONTAINER = 'document'
+process.env.AZURE_STORAGE_USE_CONNECTION_STRING = 'true'
+process.env.AZURE_STORAGE_ACCOUNT_NAME = 'test'
+process.env.AZURE_STORAGE_CONNECTION_STRING = 'test'
+process.env.COOKIE_PASSWORD = 'test-55baf113-a8dc-4957-97e7-1f5340ace375'
+process.env.TERMS_AND_CONDITIONS_URL = 'test'
 
-jest.mock('../app/config/storage', () => ({
-  storageAccount: 'mockStorageAccount'
-}))
+require('dotenv').config()
```

Comment:
Do we still want to use Dotenv here. Won't calling config() on this now override the specific env vars you've set above with any weird shenanigans you've got set in your .env file?

Reply:
Not sure if it matters, none of the ones set above need to have any specific value, I've just put 'something' in there so that they don't fail the validation inside our config generators. (though the URIs need to at least validate as urls) As before this was always stubbed it his the fact that most of these weren't set to anything in our test runner environment.

If you need one of these values to be something specific you can override via `jest.replaceProperty` in the test itself.

Have to admit I'm not a huge fan of having these just created globally but almost every test fails without them so you'd be repeating this a lot otherwise 😞

Reply:
OK, I can live with that